modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -380,29 +380,58 @@ a|Write the ISO to a new output file.
 a|`-h`, `--help`
 a|Print help information.
 
-2+|*coreos-installer PXE Ignition subcommands*
+2+|*coreos-installer PXE subcommands*
 
 |*_Subcommand_* |*_Description_*
 
 2+|Note that not all of these options are accepted by all subcommands.
 
+a|`coreos-installer pxe customize <options> <path>`
+a|Customize a {op-system} live PXE boot config.
+
 a|`coreos-installer pxe ignition wrap <options>`
 a|Wrap an Ignition config in an image.
 
 a|`coreos-installer pxe ignition unwrap <options> <image_name>`
 a|Show the wrapped Ignition config in an image.
 
-2+|*coreos-installer PXE Ignition subcommand options*
+2+|*coreos-installer PXE customize subcommand options*
 
 |*_Option_* |*_Description_*
 
 2+|Note that not all of these options are accepted by all subcommands.
 
-a|`-i`, `--ignition-file <path>`
-a|The Ignition config to be used. Default is `stdin`.
+a|`--dest-ignition <path>`
+a|Merge the specified Ignition config into a new configuration fragment for the destination system.
```

Comment:
Add "file" for clarity to "Ignition config"?

Reply:
Thanks, I've updated all instances of this to use "Ignition config file"